client/state/ui/selectors/sidebar-visibility.js

```diff
@@ -0,0 +1,3 @@
+export default function getSidebarIsCollapsed( state = false ) {
```

Comment:
Why do we pass `false` as a default here? It would result in an error if the selector logic operated on it as `false` doesn't have a property `ui`...etc.

Reply:
Good catch, copy - paste error. Fixed https://github.com/Automattic/wp-calypso/pull/46045/commits/ae043b12094c73d282e32673cafa05d52f3eba5b